extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -852,7 +851,7 @@ export class ProjectsController {
 			if (await utils.exists(absolutePath + constants.sqlFileExtension)) {
 				absolutePath += constants.sqlFileExtension;
 			} else {
-				await this.apiWrapper.showErrorMessage(constants.cannotResolvePath(absolutePath));
+				vscode.window.showErrorMessage(constants.cannotResolvePath(absolutePath));
```

Comment:
@SakshiS-harma FYI - I changed this because I don't think this is the behavior you intended. awaiting this means that it won't continue on until the user dismisses the dialog. So unless you're using the return value (asking a question) we generally shouldn't be awaiting the calls to the show message APIs

Reply:
Makes sense. Thanks for fixing this.